shared/src/main/kotlin/org/javacs/kt/util/URIs.kt

```diff
@@ -1,8 +1,9 @@
 package org.javacs.kt.util
 
-import java.nio.charset.StandardCharsets
+import org.eclipse.core.runtime.URIUtil
```

Comment:
Is there a way to get this URI conversion behavior without pulling in a new dependency on the Eclipse runtime?

Reply:
I tried to write the conversion code myself but then I realized URI are more complicated that I thought (Like Datetime :wink:). This is the source code for [`URIUtil.fromString`](https://github.com/eclipse/rt.equinox.bundles/blob/master/bundles/org.eclipse.equinox.common/src/org/eclipse/core/runtime/URIUtil.java#L128-L150).

Reply:
Are there other characters than spaces that we would have to handle? If not, we just do a replacement of spaces by `%20`.

This seems to be a [somewhat common issue](https://stackoverflow.com/questions/2593214/android-how-to-parse-url-string-with-spaces-to-uri-object).